SitemapAwareInterface.php

```diff
@@ -15,13 +15,9 @@
 /**
  * Models implementing this interface can indicate whether they should be listed in the sitemap or not.
  *
- * For PHPCR-ODM documents used with the \Symfony\Cmf\Bundle\SeoBundle\Doctrine\Phpcr\SitemapUrlInformationProvider,
- * make sure to call the property that stores this information "visible_for_sitemap",
- * as the provider uses the property in the query rather than loading all documents and checking the method.
- *
  * @author Maximilian Berghoff <Maximilian.Berghoff@gmx.de>
  */
-interface SitemapElementInterface
+interface SitemapAwareInterface
```

Comment:
should the method signature accept the sitemap name? would be the consequent thing with the rest of how sitemap is built. the document is still free to ignore the name if its a global flag.